README.md

```diff
@@ -1,20 +1,222 @@
+
+      @Entity
+      @Table(schema = "my_custom_schema", name = "legacy_replica_path",
+          uniqueConstraints = @UniqueConstraint(columnNames = { "path", "creation_timestamp" }))
+      public class MyJobsLegacyReplicaPath extends EntityLegacyReplicaPath {
+
+        protected MyJobsLegacyReplicaPath() {}
+
+        public MyJobsLegacyReplicaPath(String path) {
+          this.path = path;
+          this.eventId = "";
+          this.pathEventId = "";
+        }
+      }
+
+To accompany the custom `LegacyReplicaPath` implementation the user should extend the `LegacyReplicaPathRepository` interface providing the custom `LegacyReplicaPath` implementation as a generic type argument. This simplifies the creation of a `CrudRepository` for your `LegacyReplicaPath`.
+For example:
+
+      public interface MyJobLegacyReplicaPathRepository
+          extends LegacyReplicaPathRepository<MyJobsLegacyReplicaPath> {
+      }
+
+Whether you are using a custom housekeeping configuration, or the defaults, your application must provide two crucial annotations which will load the Entities and CrudRepositories that you require. These are the `@EntityScan` and `@EnableJpaRepositories` annotations. These annotations are best demonstrated in an example:
+
+    //The class annotated with `@Entity` that defines the required LegacyReplicaPath implementation
+    @EntityScan(basePackageClasses = HousekeepingLegacyReplicaPath.class)
+    //The class which extends LegacyReplicaPathRepository and contains your desired `CrudRepository` implementation
+    @EnableJpaRepositories(basePackageClasses = HousekeepingLegacyReplicaPathRepository.class)
+
+
+## Password Encryption
+Housekeeping allows users to provide encrypted passwords in their configuration or programs. The Housekeeping project has the jasypt-1.9.2.jar in the /lib directory. This jar can be used to generate encrypted passwords which can be decrypted by Spring Boots jasypt support in your application. The necessary dependencies for decrypting jasypt encrypted passwords are packaged with the Housekeeping project.
+
+An encrypted password can be generated by doing the following:
+
+    java -cp jasypt-1.9.2.jar  org.jasypt.intf.cli.JasyptPBEStringEncryptionCLI input="Ch4ll3ng3" password=db_password algorithm=PBEWithMD5AndDES
+
+    ----ENVIRONMENT-----------------
+
+    Runtime: Oracle Corporation OpenJDK 64-Bit Server VM 25.121-b13
+
+
+    ----ARGUMENTS-------------------
+
+    algorithm: PBEWithMD5AndDES
+    input: Ch4ll3ng3
+    password: db_password
+
+
+    ----OUTPUT----------------------
+
+    EHL/foiBKY2Ucy3oYmxdkFiXzWnOu7by
+
+The 'input' is your database password. The 'password' is a password specified by you that can be used to decrypt the data.
+The 'output' is your encrypted password. This encrypted password can then be used in the Circus Train yaml configuration:
```

Comment:
"Circus Train yaml" - no mentions of CT here please! You could say something like "This encrypted password can then be used in a YAML configuration like so"